Foundation/NSFileManager.swift

```diff
@@ -9,10 +9,16 @@
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
     import Glibc
 #endif
 
+#if os(Android) // struct stat.st_mode is UInt32
+func &( left: UInt32, right: mode_t ) -> mode_t {
```

Comment:
Best to annotate explicitly as `internal`? Also, same nit about space inside parens.